scripts/variantstore/wdl/GvsTieoutVcfMaxAltAlleles.wdl

```diff
@@ -0,0 +1,98 @@
+version 1.0
+
+import "GvsUtils.wdl" as Utils
+
+workflow GvsTieoutVcfMaxAltAlleles {
```

Comment:
Would it make sense to add this to a test case? Maybe run the workflow with maximum_alt_alleles defined as some largeish number?